pt-BR/mail/chrome/messenger/AccountManager.dtd

```diff
@@ -2,13 +2,13 @@
 
 <!ENTITY accountManagerTitle.label "Configurar contas">
 
-<!ENTITY accountActionsButton.label "Account Actions">
+<!ENTITY accountActionsButton.label "Ações da conta">
 <!ENTITY accountActionsButton.accesskey "A">
-<!ENTITY addMailAccountButton.label "Add Mail Account…">
+<!ENTITY addMailAccountButton.label "Adicionar conta de e-mail…">
```

Comment:
I disagree. E-mail is absolutely fine, and it's shorter, which is important when we're dealing with UIs that have limited space.